### PR TITLE
Add @types/circuit-breaker-js and adjust client.ts accordingly

### PR DESCRIPTION
**Pull request by DeTeam**
Now we have this https://www.npmjs.com/package/@types/circuit-breaker-js — so updating some types

### CODE REVIEW

**Comment by shuhei** [comment]
:+1:

File: lib/client.ts [hunk 1]
```diff
@@ -217,12 +217,18 @@ const requestWithFilters = (client: ServiceClient, params: ServiceClientRequestO
 }
 
 const noop = () => {/* do nothing */ }
-const noopBreaker = { run (command: Function) { command(noop, noop) } }
+const noopBreaker: CircuitBreaker = {
+  run (command) { command(noop, noop) },
+  forceClose: () => null,
+  forceOpen: () => null,
+  unforce: () => null,
+  isOpen: () => true
```
Comment (ruiaraujo) [inline lib/client.ts]: nit true is wierd here.

Comment (DeTeam) [reply lib/client.ts]: lol, it should be `false`, thanks :)

Comment (DeTeam) [reply lib/client.ts]: Pushed the change.

**Comment by DeTeam** [comment]
👍

**Comment by ruiaraujo** [comment]
👍

**Pull request merged**